test/SIL/Serialization/opaque_return_type_equality.swift

```diff
@@ -0,0 +1,13 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend -swift-version 5 -disable-availability-checking -c %S/Inputs/opaque_return_type_equality_input.swift -emit-module-path %t/opaque_return_type_equality_input.swiftmodule -I %t -O -module-name opaque_return_type_equality_input -o %t/opaque_return_type_equality_input.o
+// RUN: %target-swift-frontend -swift-version 5 -disable-availability-checking -c %S/opaque_return_type_equality.swift -emit-module-path %t/opaque_return_type_equality.swiftmodule -I %t -O -module-name opaque_return_type_equality -o %t/opaque_return_type_equality.o
+
+// Check that the SIL type equality check asserts don't fail between the opaque return types and their underlying types.
```

Comment:
Does this not require an additional flag to be passed? Something along the lines of `-enable-sil-verify-all` or `-sil-verify-all`.

Reply:
I was able to trigger the assert failure without one.